src/structures/CommandInteraction.js

```diff
@@ -92,8 +92,8 @@ class CommandInteraction extends Interaction {
    * @property {CommandInteractionOption[]} [options] Additional options if this option is a
    * subcommand (group)
    * @property {User} [user] The resolved user
-   * @property {GuildMember|APIGuildMember} [member] The resolved member
-   * @property {GuildChannel|APIChannel} [channel] The resolved channel
+   * @property {GuildMember|APIInteractionDataResolvedOption} [member] The resolved member
```

Comment:
This isn't really... accurate, or *strict*. it's... fine probably, but idk. @discordjs/the-big-3 thoughts?

Reply:
There’s no better way to indicate that that is the data that you can receive though

Reply:
Yea there is.. there's discord-api-types for it too 👀

Reply:
But how do you do that with documentation?

Reply:
@iCrawl might be better to ask here, but you could just have links to them like how it was before. Or point to the raw structures in Discord's documentation 🤷‍♂️

Reply:
Nah, I think its fine.